olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationSettings.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include <boost/optional.hpp>
+#include <olp/core/http/NetworkProxySettings.h>
+#include <olp/authentication/AuthenticationApi.h>
+
+namespace olp {
+namespace http {
+class Network;
+}
+
+namespace thread {
+class TaskScheduler;
+}
+
+namespace authentication {
+/// The default token endpoint url.
+static constexpr auto kHereAccountProductionTokenEndpointUrl =
+    "https://account.api.here.com/oauth2/token";
```

Comment:
``` suggestion
static constexpr auto kHereAccountProductionTokenEndpointUrl =
    "https://account.api.here.com";
```
In current implementation, AuthenticationClient has url input which is Settings::token_endpoint_url but without "/oauth2/token" ending. Maybe this default value should be without the ending too, so it could be used instead of the url?

Reply:
Yeah, this is a long story but currently, it is still necessary but we can adapt later on during the refactoring of the TokenEndpoint.